From: 13737698565 <2778130856@qq.com>
Date: Sun, 6 Apr 2025 09:07:40 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 2.3% (1 of 43 strings)

Translation: openkylin-nile-new/kylin-os-manager-win-data-migration
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-win-data-migration/vi/
---
 treasure-chest/win-data-migration/translations/win-data-migration_vi.ts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/treasure-chest/win-data-migration/translations/win-data-migration_vi.ts b/treasure-chest/win-data-migration/translations/win-data-migration_vi.ts
index 9c7aa9a..7054d94 100644
--- a/treasure-chest/win-data-migration/translations/win-data-migration_vi.ts
+++ b/treasure-chest/win-data-migration/translations/win-data-migration_vi.ts
@@ -46,7 +46,7 @@
     </message>
     <message>
         <source>ok</source>
-        <translation type="unfinished"></translation>
+        <translation>Đồng ý</translation>
     </message>
     <message>
         <source>cancel</source>
